### PR TITLE
Build aarch64-pc-windows-msvc releases

### PR DESCRIPTION
**Pull request by lnicola**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -45,11 +45,41 @@ jobs:
...
+        name: x86_64-pc-windows-msvc
         path: ./dist
 
-  dist-ubuntu:
-    name: dist (Ubuntu 16.04)
+  dist-aarch64-pc-windows-msvc:
+    name: dist (aarch64-pc-windows-msvc)
+    runs-on: windows-latest
+    env:
+      RA_TARGET: aarch64-pc-windows-msvc
```
Comment (matklad) [inline .github/workflows/release.yaml:55]: wow, one can just cross-compile msvc? nifty!

**Review by matklad — COMMENTED** [review]
does this need a corresponding .ts change?

**Comment by lnicola** [comment]
It does, but I'd rather do a different PR.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (lnicola) [reply .github/workflows/release.yaml:55]: I'm still waiting for a test run, but it works locally if you install the platform-specific build tools.

**Comment by matklad** [comment]
r=me

**Comment by lnicola** [comment]
Well, this is new:

image

What better way to start a week than breaking the nightly builds?

bors r=matklad

**Pull request merged**